protocol/proofs/chain-config-registry.md

```diff
@@ -0,0 +1,94 @@
+# Purpose
+
+Design a solution that makes it easier to add or update chain configurations for use in fault proof programs and
+separate the release of op-program from updates to chain configuration.
+
+# Problem Statement + Context
+
+Currently, op-program and kona embed the configurations for chains they support via the superchain-registry and these
+are committed to on-chain as part of the `FaultDisputeGame` absolute pre-state. For existing chains that activate hard
+forks as part of the superchain configuration, this works well and the new chain configuration is included in the same
+update to the absolute prestate that implements the new hard fork rules in the fault proof program.
+
+However, chains that activate hard forks after the initial superchain set or new chains deployed between hard forks,
+they are unable to use the same absolute prestate because it doesn't contain the latest chain configuration. A new
```

Comment:
nit: 
```suggestion
However, chains that activate hard forks after the initial superchain set or new chains deployed between hard forks,
are unable to use the same absolute prestate because it doesn't contain the latest chain configuration. A new
```